env-dist

```diff
@@ -1,7 +1,7 @@
 SECRET_KEY = 'secret_key'
 # Site settings
 SITE_URL = http://127.0.0.1:8000
-ALLOWED_HOSTS = 127.0.0.1
+ALLOWED_HOSTS = 127.0.0.1, .localhost
```

Comment:
maybe a stupid question, but why does that need to be with a dot for localhost?

Reply:
Because this will allow sub-domains for local dev in a multi-site setup. Eg site1.locahost, site2.localhost. Good question!

Reply:
oh sure, interesting approach :)